docs/source/schemas/index.rst

```diff
@@ -133,33 +133,35 @@ characteristics, among others. The types are:
 +-------------------------------+--------------+---------------+----------------------+----------------+
 | NULLABLE\_FIXED\_LONG         | long         | byte[9]       | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_LONG                     | long         | byte[len]\*   | YES                  | YES            |
+| VAR\_LONG                     | long         | byte[len] \*  | YES                  | YES            |
++-------------------------------+--------------+---------------+----------------------+----------------+
+| VAR\_SIGNED\_LONG             | long         | byte[len] \*  | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_SIGNED\_LONG             | long         | byte[len\*]   | YES                  | YES            |
+| UUID                          | UUID         | byte[16]      | YES                  | YES            |
```

Comment:
should we spec the encoding of UUID (and other types) as they are all stored in big endian order?